honeybee_energy/hvac/allair/_base.py

```diff
@@ -63,12 +63,13 @@ class _AllAirBase(_TemplateSystem):
         * latent_heat_recovery
         * demand_controlled_ventilation
         * schedules
+        * user_data
     """
     __slots__ = ('_economizer_type', '_sensible_heat_recovery', '_latent_heat_recovery',
-                 '_demand_controlled_ventilation')
+                 '_demand_controlled_ventilation', '_user_data')
```

Comment:
This is not the correct way to use `__slots__`. If the variable is already defined in the class that you are inheriting from, you should not repeat it here. Otherwise you are un-doing a lot of the memory saving that we get from using `__slots__`. So you should remove this from here and from the `__slots___` of all other HVAC system classes. More info is here:

https://stackoverflow.com/questions/1816483/how-does-inheritance-of-slots-in-subclasses-actually-work

Reply:
AHHHHH Okay cool thaat made sense of ```__slots__```  ty!! commit: allair slots edit